recipe/meta.yaml

```diff
@@ -11,12 +12,17 @@ source:
   sha256: 724f3b430dca94733aea8325cdf5a9a2043184c3bc7dff8cf6452e1848efd4e3
 
 build:
-  number: 0
+  number: {{ build_number }}
+  ignore_run_exports:
+    - cudatoolkit
```

Comment:
Is there some reason this is being dropped from `requirements/run`?

Reply:
I just copied that from the [meta.yaml](https://github.com/conda-forge/parallelproj-feedstock/blob/main/recipe/meta.yaml) of parallelproj. At the time when I submitted the staged recipe of parallelproj, this line was added by someone from the conda-forge team. If I remember correctly, it was @carterbox TBH I don't remember why it can be ignored.

Reply:
If we don't know why `cudatoolkit` can be ignored, we probably shouldn't be ignoring it

That said, am happy to wait for Daniel (or someone) to provide more context on why it was ignored `parallelproj` (though the reasoning may be different for this recipe)

Reply:
STIR itself contains no cuda code. So if we can ignore `cudatoolkit` for parallelproj, we can ignore it for STIR. 
The relevant part of the CMakeList.txt that builds the cuda lib of parallelproj is [here](https://github.com/gschramm/parallelproj/blob/5c37b2cb3529e137acbe9077ea62ca13062b79b3/cuda/CMakeLists.txt#L49).

Reply:
Does it make sense to have a CUDA build then if the build wouldn't change? Or does `stir` link to different libraries when building with a CUDA-enabled `parallelproj`?

Reply:
Indeed. If cuda is there (and the cuda version of parallelproj), STIR links against `libparallelproj_cuda.so` instead of l`ibparallelproj_c.so`

Reply:
Not every package which uses the cuda compiler links against the cudatoolkit libraries. On a per-package basis, you can inspect the build logs to see whether any libraries from the cudatoolkit are used.

Reply:
I just double checked the last [build log of parallelproj](https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/659256/logs/82):
```
2023-02-16T03:09:13.7255923Z [23/26] : && $BUILD_PREFIX/bin/x86_64-conda-linux-gnu-c++ -fPIC -Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,--allow-shlib-undefined -Wl,-rpath,$PREFIX/lib -Wl,-rpath-link,$PREFIX/lib -L$PREFIX/lib -shared -Wl,-soname,libparallelproj_cuda.so.1.2.14 -o cuda/libparallelproj_cuda.so.1.2.14 cuda/CMakeFiles/parallelproj_cuda.dir/src/joseph3d_back_cuda.cu.o cuda/CMakeFiles/parallelproj_cuda.dir/src/joseph3d_back_tof_lm_cuda.cu.o cuda/CMakeFiles/parallelproj_cuda.dir/src/joseph3d_back_tof_sino_cuda.cu.o cuda/CMakeFiles/parallelproj_cuda.dir/src/joseph3d_fwd_cuda.cu.o cuda/CMakeFiles/parallelproj_cuda.dir/src/joseph3d_fwd_tof_lm_cuda.cu.o cuda/CMakeFiles/parallelproj_cuda.dir/src/joseph3d_fwd_tof_sino_cuda.cu.o cuda/CMakeFiles/parallelproj_cuda.dir/src/projector_kernels.cu.o cuda/CMakeFiles/parallelproj_cuda.dir/src/utils_cuda.cu.o CMakeFiles/parallelproj_cuda.dir/cmake_device_link.o  $PREFIX/lib/libgomp.so  -lpthread  -lcudadevrt  -lcudart_static  -lrt  -lpthread  -ldl -L"/usr/local/cuda/targets/x86_64-linux/lib/stubs" -L"/usr/local/cuda/targets/x86_64-linux/lib" && :
```

In there, we have `-lcudadevrt  -lcudart_static`. @carterbox we don't need the cudatoolkit for those (runtime) cuda libs?

Reply:
Yeah CUDART is being statically linked and `cudadevrt` is only statically linked. So as long as no other CUDA libraries are used, it should be ok to ignore

To Daniel's point above, would look at the lines after `Packaging parallelproj`. These list a number of binaries in the package and what libraries are linking to. This will provide clues as to what libraries would be expected at runtime. It should also raise warnings (or errors) about missing libraries at runtime, which would be the thing to look for